src/main/drivers/serial_uart_stm32f4xx.c

```diff
@@ -264,10 +264,7 @@ static void handleUsartTxDma(uartPort_t *s)
 {
     DMA_Cmd(s->txDMAStream, DISABLE);
 
```

Comment:
For F4, it's get cleared in `dmaIRQHandler`
https://github.com/jflyper/cleanflight/blob/9a62a678e64e14050360e9aa862012622b1501d9/src/main/drivers/serial_uart_stm32f4xx.c#L270-L291